### PR TITLE
Validate admins for the current BOSH

### PR DESCRIPTION
**Pull request by pburkholder**
## Changes proposed in this pull request:
-
- gets the correct secret for Tooling and PRODUCTION.
-

## security considerations

Improvement - makes it easier to check admins in our envs

### CODE REVIEW

File: validate-admins.sh [hunk 1]
```diff
@@ -1,38 +1,57 @@
...
+        uaac users -a username "id eq '${uuid}'" | grep username | awk '{print $2}'
+    done
+
+    echo -e "\n\nadmins for concourse\n"
+    for uuid in $(uaac groups -a members "displayName eq 'concourse.admin'" | grep value | awk '{print $2}'); do 
+        uaac users -a username "id eq '${uuid}'" | grep username | awk '{print $2}'
+    done
+}
+
+case "$BOSH_DIRECTOR_NAME" in
```
Comment (bengerman13) [inline validate-admins.sh:31]: two things, each maybe more work than they're worth:
- we could lowercase the director name and use it in the command to get the secret
- we could fetch the uaa url from credhub

File: validate-admins.sh [hunk 2]
```diff
@@ -1,38 +1,57 @@
...
+    secret=$(credhub get -n /toolingbosh/opsuaa/uaa_admin_client_secret | grep value | sed -r 's/value: //g')
+    main opsuaa.fr.cloud.gov $secret
+    ;;
+  *)
+    if [ "$#" -ne 2 ]; then
+    echo
+    echo "Usage:"
+    echo "   ./validate-admins.sh <uaa-target> <uaa-admin-client-secret>"
+    echo
+    echo "   EX:  ./validate-admins.sh login.fr.cloud.gov S3c4Et"
```
Comment (bengerman13) [inline validate-admins.sh:46]: this example says login.fr.cloud.gov - should we be using login on line 34? or is the example wrong?

**Review by bengerman13 — APPROVED** [review]
left a few comments for consideration, but it's fine as-is :)

File: validate-admins.sh (hunk 1 above)
Comment (pburkholder) [reply validate-admins.sh:31]: I don't know how to do those, so leaving as is.

**Pull request merged**